lib/periodic_jobs.rb

```diff
@@ -0,0 +1,84 @@
+  # Clear out EVSS disability claims that have not been updated in 24 hours
+  mgr.register('20 2 * * *', 'DeleteOldPiiLogsJob')
+  # Clear out old personal information logs
+  mgr.register('30 2 * * *', 'CentralMail::DeleteOldClaims')
+  # Clear out central mail claims older than 2 months
+  mgr.register('0 2 1 * *', 'VBADocuments::ReportMonthlySubmissions')
+  # Monthly report of benefits intake submissions
+
+  mgr.register('0 3 * * *', 'DeleteOldTransactionsJob')
+  # Deletes old, completed AsyncTransaction records
+  mgr.register('0 3 * * 1-5', 'EducationForm::CreateDailySpoolFiles')
+  # Generate a daily set of spool files and transmit them over SFTP to a backend system
+  mgr.register('30 3 * * 1', 'EVSS::FailedClaimsReport')
+  # Notify developers about EVSS claims which could not be uploaded
+
+  mgr.register('0 4 * * *', 'EducationForm::CreateDailyFiscalYearToDateReport')
+  # Send the daily report to VA stakeholders about Education Benefits submissions
+  mgr.register('5 4 * * 1-5', 'EducationForm::CreateSpoolSubmissionsReport')
+  # Send the daily report to the call center about spool file submissions
+  mgr.register('10 4 * * *', 'Facilities::DentalServiceReloadJob')
+  # Download and cache facility access-to-care metric data
+  mgr.register('25 4 * * *', 'Facilities::MentalHealthReloadJob')
+  # Download and cache facility mental health phone number data
+  mgr.register('35 4 * * 1-5', 'EducationForm::Create10203SpoolSubmissionsReport')
+  # Send the daily 10203 report to the call center about spool file submissions
+  mgr.register('45 4 * * *', 'Facilities::AccessDataDownload')
+  # Download and cache facility access-to-care metric data
+  mgr.register('55 4 * * *', 'Facilities::PSSGDownload')
+  # Download and store drive time bands
+  mgr.register('0 5 * * 1', 'AppealsApi::WeeklyErrorReport')
+  mgr.register('0 6 * * *', 'AccountLoginStatisticsJob')
+  # Gather account login statistics for statsd
+  mgr.register('0 6-18/6 * * *', 'EducationForm::Process10203Submissions')
+  # Go through 22-10203 determine if application should be processed as part of normal submission process or rejected
+  mgr.register('0 8 * * 1-5', 'AppealsApi::DailyStuckRecordsReport')
+  # Daily report of all stuck appeals submissions
+  mgr.register('0 12 3 * *', 'CypressViewportUpdater::UpdateCypressViewportsJob')
+  # Updates Cypress files in vets-website with data from Google Analytics.
+  mgr.register('0 13 * * 1', 'Mobile::V0::WeeklyMaintenanceWindowLogger')
+  # Weekly logs of maintenance windows
+  mgr.register('0 20 * * *', 'ClaimsApi::ClaimAuditor')
+  # Daily alert of pending claims longer than acceptable threshold
+  mgr.register('0 23 * * 1-5', 'AppealsApi::DecisionReviewReportDaily')
+  # Daily report of appeals submissions
+  mgr.register('0 23 * * 7', 'AppealsApi::DecisionReviewReportWeekly')
+  # Weekly report of appeals submissions
+  mgr.register('30 23 * * 1-5', 'AppealsApi::DailyErrorReport')
```

Comment:
This one is `0 23 * * MON-FRI` in sidekiq_scheduler

Reply:
There are a lot of jobs scheduled at the same time. During these times, I've noticed a higher amount of errors. Spreading out the jobs a tiny bit to reduce the load.